### PR TITLE
Spanish translation of all content

### PR DESCRIPTION
**Pull request by onavascuez**
- :es: Translation of all content
- Added one string on sidebar

### CODE REVIEW

**Comment by onavascuez** [comment]
On time added all translations, i think its necessary make one more insert on languages table, to add this new language. :)

File: app/Modules/Auth/Resources/Lang/es/main.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    'new_pw'            => 'Nueva contraseña',
+    'email_new_pw'      => 'Te hemos enviado a tu correo electrónico tu nueva contraseña.',
+    'pw_link'           => 'Por favor, haz click en el siguiente enlace para generar una nueva contraseña para :0.',
+    'email_ignore'      => 'Si no quieres generar una nueva contraseña, por favor, ignora este correo eletrónico.',
+    'pw_generated'      => 'Se ha generado una nueva contraseña para :0.',
+
+    /* 
+     * Login 
+     */
+    'login'             => 'Login',
```
Comment (frandominguezl) [inline app/Modules/Auth/Resources/Lang/es/main.php]: Better 'Inicar Sesión' maybe?

Comment (onavascuez) [reply app/Modules/Auth/Resources/Lang/es/main.php]: Hey! You right! I'm so used to reading "login" that I overlooked at! Thanks!

**Comment by chriskonnertz** [comment]
Hello,

wow, first of all thank you for doing all this translation work! Great job. I am sure many Spanish users will like this a lot.

I want to merge it but not into the 2.4 branch, because development of version 2.4 is complete and such  a big feature should go into the 2.5 version/branch. I am not sure what happens when I change the branch so maybe it's the best when you do it?

**Review by frandominguezl — APPROVED** [review]

**Comment by oriolwonga** [comment]
Hey Chris! Thanks!
I try to download this week, or next, and install the 2.5, and move all translations to this version, and addapt this to it. No problem!

I got this!

**Comment by onavascuez** [comment]
Hey Chris! Thanks!
I try to download this week, or next, and install the 2.5, and move all translations to this version, and addapt this to it. No problem!

I got it!

**Comment by chriskonnertz** [comment]
Do we still need this pull request? is something from this 2.5 pull request missing? https://github.com/Contentify/Contentify/pull/428

**Comment by chriskonnertz** [comment]
I think everything is included in the 2.6-dev branch.
There is no diff between the code of your pull request and 2.6-dev: https://github.com/Contentify/Contentify/compare/2.6-dev...onavascuez:onavascuez-es-lang

(Just for your information: It does not make sense to merge it into the 2.4 branch, because the 2.4 branch is no longer in use. Actually I plan to delete the 2.4 branch. The code of the 2.4 branch would still exist in the 2.4 **tag**, but the tag is immutable so we cannot bring your changes to the tag.)

**Comment by onavascuez** [comment]
> Do we still need this pull request? is something from this 2.5 pull request missing? #428

No, you can delete it. :)

**Comment by chriskonnertz** [comment]
Ok, closing it now.

**Pull request closed without merging**